concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1341,7 +1341,6 @@ jobs:
         - get: bosh-secrets
         - get: concourse-manifest
           passed: ['generate-concourse-config']
-        - get: concourse-secrets
```

Comment:
Good catch.

Reply:
I didn't catch it, our concourse linter did...